src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -427,8 +457,13 @@ class editIndex extends viewModelBase {
         const reduceFromPreview = previewItem.Definition.Reduce;
         
         const newIndexDefinition = new indexDefinition(this.editedIndex().toDto());
+        
```

Comment:
do we need `if (!this.isEditingExistingIndex()) {` in this method?

Reply:
refactored this area a bit
but yes - need this method to know whether this is a clone or not